modules/async/async/shared/src/main/scala/async.scala

```diff
@@ -42,5 +53,18 @@ object async {
       }
     }
   }
+
+  trait Syntax {
+
+    implicit def futureOps[A](f: Future[A]): FutureOps[A] = new FutureOps(f)
+
+    final class FutureOps[A](f: Future[A]) {
+
+      def to[F[_]](implicit AC: AsyncContext[F], E: ExecutionContext): F[A] = future2AsyncM[F, A](f)
```

Comment:
Personally I think I would call this `unsafeTo`, but the `AsyncM` type class has no real "suspension" guarantees.
We could implement it more safely for `Eval[Future[A]]` or a method which takes `=> Future[A]`.

We just have to remember that when we want to end up in something like monix `Task` or `cats.effect.IO`, this `to` method is probably not the best route.

Reply:
Sounds good, I've just deprecated it in https://github.com/frees-io/freestyle/pull/507/commits/9acae08d059f3cea1364018bfec7ef3e53936fd2

Reply:
Sounds good. Created #509